spec/integration/post_spec.rb

```diff
@@ -592,6 +592,26 @@
         expect(status.description).to eq(custom_status[:description])
         expect(status.emoji).to eq(custom_status[:emoji])
       end
+
+      context "when custom emoji is set" do
+        custom_emoji = "palm_tree"
+
+        before { SiteSetting.holiday_status_emoji = custom_emoji }
+
+        it "sets holiday user status with custom emoji" do
```

Comment:
Can we add a test case here for when `holiday_status_emoji` is blank? I see that it will default to the date emoji, but a test case would be good just to prevent regressions in the future.